orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -6496,3 +6496,39 @@ def run_step(model, x, y, z):
     torch.cuda.synchronize()
     if original_val is not None:
         os.environ["ORTMODULE_MEMORY_OPT_LEVEL"] = original_val
+
+
+def test_bert_memory_inspection(caplog):
+    original_val = os.environ.get("ORTMODULE_PRINT_MEMORY_STATS", None)
```

Comment:
Not sure if this env var is really set during testing.

Reply:
yeah, nobody actually set it unless we want to enable that locally when running tests. This is a common approach to handle the env var in the test, so this is to follow that. :)